### PR TITLE
create tab home UI

### PR DESCRIPTION
**Pull request by QuangHieuu**
task: Create UI tab home
https://edu-redmine.framgia.vn/issues/2654
screenshot_20181016-095116

### CODE REVIEW

**Comment by QuangHieuu** [comment]
ready

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
         tools:ignore="GoogleAppIndexingWarning">
-        <activity android:name="com.framgia.music_40.MainActivity">
+        <activity android:name="com.framgia.music_40.screen.main.MainActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN"/>
```
Comment (phuongohn-1626) [inline app/src/main/AndroidManifest.xml:19]: Thêm screenOrientation

File: app/src/main/java/com/framgia/music_40/screen/main/homeScreen/HomeScreen.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+        RecyclerView recyclerView = view.findViewById(R.id.recycler_view_home_screen);
+        recyclerView.setAdapter(new HomeScreenAdapter(getContext(), getGenresList()));
+
+        return view;
+    }
+
+    private List<Genres> getGenresList() {
+        List<Genres> list = new ArrayList<>();
+        list.add(new Genres("All Music", "all_music"));
```
Comment (phuongohn-1626) [inline app/src/main/java/com/framgia/music_40/screen/main/homeScreen/HomeScreen.java]: hard code :))

File: app/src/main/java/com/framgia/music_40/screen/main/homeScreen/HomeScreenAdapter.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    class ViewHolder extends RecyclerView.ViewHolder {
+
+        private TextView mTextView;
+        private ImageView mImageView;
+
+        ViewHolder(@NonNull View itemView) {
+            super(itemView);
+
```
Comment (phuongohn-1626) [inline app/src/main/java/com/framgia/music_40/screen/main/homeScreen/HomeScreenAdapter.java]: remove blank

**Comment by QuangHieuu** [comment]
ready

**Comment by QuangHieuu** [comment]
ready

**Review by phuongohn-1626 — APPROVED** [review]

**Comment by QuangHieuu** [comment]
ready

**Comment by QuangHieuu** [comment]
ready

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -5,24 +5,33 @@
...
         app:layout_constraintBottom_toTopOf="@id/navigation_bottom"
-        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/toolbar"
         />
 
     <android.support.design.widget.BottomNavigationView
         android:id="@+id/navigation_bottom"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="40dp"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_main.xml]: hard code

**Comment by tuanlvt-1117** [comment]
commented

**Comment by QuangHieuu** [comment]
ready

**Comment by QuangHieuu** [comment]
ready

**Comment by QuangHieuu** [comment]
ready

**Comment by QuangHieuu** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Comment by tuanlvt-1117** [comment]
merged

**Pull request merged**